app/products/product_info.py

```diff
@@ -16,6 +16,8 @@ def create_match_paterns(self):
         self.price_pattern = re.compile(
             r"(price|cost|how much)", re.IGNORECASE)
         self.stock_pattern = re.compile(r"(stock|how many)", re.IGNORECASE)
+        self.nutrition_pattern = re.compile(
+            r"(calories|protein|carbs|carbohydrates|sugar|fat|nutrition|nutritional)", re.IGNORECASE)
```

Comment:
A nice to have is to handle questions like "Does this bread make me gain weight?"